tests/test_inner_loop.py

```
@@ -260,6 +270,129 @@ def test_handle_waiting_on_review_state_missing_pr_forces_needs_input(
+            number=pr.number,
+            repo=pr.repo,
+            review_status="approved",
+            ci_status="success",
+            merged_at=None,
+            last_checked_at="2026-02-09T00:00:01Z",
+        ),
+    )
+    control = inner_loop_module.LoopControlState(backoff_seconds=5.0)
+    evaluations = {"count": 0}
+
+    def fake_run_auto_approve_eval(*, run_record, runtime, control):
```

Ruff (0.15.2)

[warning] Unused function argument: `control`

(ARG001)